### DISCUSSION TITLE
GoPro12 and Zeapon

### DISCUSSION THREAD

**Discussion by lissker** [gopro/labs#1035] [Q&A]
I am the owner of a Zeapon system that can shoot very beautiful timelapses. But I did not find a solution for releasing the shutter (trigger) through the program from the GoPro and Zeapon manufacturer 

Working principle of Zeapon:
The software from the smartphone sends a command to the slider, rotating heads or timelapse triger to release the shutter, which send the signal via a standard remote cable jack 2.5 to the camera 

Is it possible to make a cable for gopro12 yourself through which the signal from Zeapon will be transmitted and so that power can be supplied through this wire for long-term filming?

![photo_2024-08-22_16-53-41](https://github.com/user-attachments/assets/184a32dc-cbfd-40b8-887f-85f1522c775d)
![photo_2024-08-22_16-53-45](https://github.com/user-attachments/assets/e11f3417-3576-4922-87f7-989ef7fcac91)

**Comment by dnewman-gpsw** [comment]
GoPro doesn't have a wired trigger, as it's typically more difficult to with a waterproof camera. Instead, triggers like this are typically done through BLE.  As Zeapon likely hasn't implemented GoPro camera control, you might need to modify a wireless remote like these https://gopro.com/en/us/shop/search?term=Remote or any of the third party solutions.

**Reply by lissker** [reply]
i thought about it and it would even be the right thing to do than to sit and solder my cable
but the wireless remote control for GoPro doesn't have a shutter release signal through the cable so i can use it with Zeapon

**Reply by dnewman-gpsw** [reply]
Shutter release cables are signal low (typically), you would use that to fire a relay that shorts the button on the remote.
